### PR TITLE
cli: Throw error if non org is provided to create-github-app

### PR DESCRIPTION
**Pull request by jhaals**

### CODE REVIEW

File: packages/cli/src/commands/create-github-app/index.ts [hunk 1]
```diff
@@ -47,3 +49,14 @@ integrations:
...
   );
 };
+
+async function verifyGithubOrg(org: string): Promise<void> {
+  const response = await fetch(
+    `https://api.github.com/orgs/${encodeURIComponent(org)}`,
+  );
+  if (!response.ok) {
+    throw new Error(
+      `GitHub organization '${org}' does not exist. Please verify the name and try again.`,
```
Comment (Rugvip) [inline packages/cli/src/commands/create-github-app/index.ts:59]: Perhaps check that we're getting the expected error code here? Thinking we'd let this through if there's a network issue, perhaps with a warning

**Review by Rugvip — COMMENTED** [review]
🎉

**Review by freben — APPROVED** [review]
Nice!

File: packages/cli/src/commands/create-github-app/index.ts (hunk 1 above)
Comment (jhaals) [reply packages/cli/src/commands/create-github-app/index.ts:71]: I tweaked it slightly, PTAL

**Pull request merged**